flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/StreamCheckpointCoordinator.scala

```diff
@@ -82,15 +81,18 @@ class StreamCheckpointCoordinator(val executionGraph: ExecutionGraph,
     case BarrierTimeout =>
       executionGraph.getState match {
         case FAILED | CANCELED | FINISHED =>
-          log.info("Stopping monitor for terminated job {}", executionGraph.getJobID)
+          log.info("[FT-Monitor] Stopping monitor for terminated job {}", executionGraph.getJobID)
           self ! PoisonPill
```

Comment:
indeed good point, I removed the tags. They had helped me filtering together messages during debugging especially in cases where there was logic involved outside the coordinator.